projects/flow.groups-2D-random-walk/src/project.py

```diff
@@ -0,0 +1,152 @@
+import flow
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# Helper functions
+def get_first_replica(jobs, first_index=1):
```

Comment:
"First" is overloaded here -- first in the iterable of jobs, or first by index?
```suggestion
def get_replica_index(jobs, replica):
```